### DISCUSSION TITLE
User Confirm Email Event

### DISCUSSION THREAD

**Discussion by davidpuplava** [OrchardCMS/OrchardCore#13688] [Ideas]
I'd like to trigger a workflow when a user has confirmed their email address (currently using TheComingSoon theme). For example, sending a "Welcome" email.

I thought about using the Http Request to identify the "ConfirmEmail" URL clicked from the email confirmation email. But I think a "User Confirmed Email" event would be more elegant.

I think I can implement this and submit a PR but wanted to open a discussion first to make sure there isn't already a better way to do this.

**Comment by hishamco** [comment]
I'm not sure we have something already OOB

**Comment by davidpuplava** [comment]
For anyone that finds this discussion thread, the UserConfirmed event was added to v2.0.0 and can be used to trigger a Welcome email: https://docs.orchardcore.net/en/latest/releases/2.0.0/#users-module
